packages/core/src/baseCogniteClient.ts

```diff
@@ -23,12 +24,19 @@ import {
   OnTokens,
 } from './login';
 import { MetadataMap } from './metadata';
-import { getBaseUrl, isUsingSSL, projectUrl } from './utils';
+import {
+  getBaseUrl,
+  isOAuthWithAADOptions,
```

Comment:
Can you expose which flow is in use to the outside? It would be convenient to expose this state our of the SDK since some APIs work with legacy auth and not native tokens, and vica verca.

Reply:
yep, definitely make sense

Reply:
`getOAuthFlowType` has been added